Client/Assets/NightModeHelper.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+  if (!window.__firefox__) {
+    window.__firefox__ = {};
+  }
+  var isEnabled = false;
+
+  function initializeStyleSheet (enabled) {
+    var night_css_id = "iBrowser_night_css";
+    var night_css = ':not(.UIButton){color: #AAAAAA !important; background-color:#2E2E2E !important;}body{-webkit-filter:brightness(.5) !important;}';
```

Comment:
I'd really prefer it if we did not special case individual websites, or even common frameworks.

We should try and generalize this – it may well be useful to other sites.

If we can't generalize this, then raise a bug, and attach the html and css.

If you want some help generalizing this, raise a bug with the html and css of a taobao page attached.